versioned_docs/version-3.x/reference/plugins/policy.md

```
@@ -0,0 +1,134 @@
+    Defines an access policy that allows a set of operations when the given condition is true.
+     - `operation`: comma-separated list of "create", "read", "update", "post-update", "delete". Use "all" to denote all operations.
+     - `condition`: a boolean expression that controls if the operation should be allowed.
+  
+- `@@deny`
+  
+    ```zmodel
+    attribute @@deny(_ operation: String, _ condition: Boolean)
+    ```
+
+    Defines an access policy that denies a set of operations when the given condition is true.
+     - `operation`: comma-separated list of "create", "read", "update", "post-update", "delete". Use "all" to denote all operations.
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 5

(MD007, ul-indent)